vcxagency-node/src/index.js

```diff
@@ -121,10 +123,15 @@ validateAppConfig(appConfig, (err, ok) => {
     appAgent.use('/agency/msg', appAgentMsg)
     appAgent.use('/', appAgentJson)
 
-    appAgent.listen(
-      appConfig.SERVER_PORT,
-      () => logger.info(`Agency is listening on port ${appConfig.SERVER_PORT}!`)
-    )
+    if (appConfig.SERVER_ENABLE_TLS === 'true') {
+      https.createServer({
+        cert: fs.readFileSync(appConfig.CERTIFICATE_PATH),
+        key: fs.readFileSync(appConfig.CERTIFICATE_KEY_PATH),
+        ca: appConfig.CERTIFICATE_AUTHORITY_PATH ? fs.readFileSync(appConfig.CERTIFICATE_AUTHORITY_PATH) : ''
```

Comment:
Is specifying empty string valid value for `ca` option?

Reply:
Yes, it is (tested).